### PR TITLE
Added a note about removal of DarkSky binding due to EOL of their API

### PR DESCRIPTION
**Pull request by cweitkamp**
- Added a note about removal of DarkSky binding due to EOL of their API

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

**Comment by wborn** [comment]
Depends on https://github.com/openhab/openhab-addons/pull/13037

**Review by wborn — CHANGES_REQUESTED** [review]
Now that the other PR got merged, can you rebase this one and add it to a `[4.0.0]` section?

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -75,6 +75,9 @@ ALERT;RFXCOM Binding: The channel 'fanSpeed' of 'fan_lucci_dc' and 'fan_novy' ha
 ALERT;TapoControl Binding: L510_Series and L530_Series Things were renamed to L510 and L530 because of manufacturer changed naming with new HW-rev. You may need to remove and create again these things.
 ALERT;WLED Binding: Binding now uses Bridge and Thing structure. Delete and add a new bridge, and at least 1 thing for each segment you require.
 
+[4.0.0]
+ALERT;Dark Sky: Due to EOL of their API, this add-on has been removed.
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:79]: There's still a merge conflict. 

Adding "Binding" would also be nice:

```suggestion
ALERT;Dark Sky Binding: Due to EOL of their API, this add-on has been removed.
```

**Comment by cweitkamp** [comment]
> There's still a merge conflict.

Sry, didn't notice. Looks like you already rebased and applied your comment. Thanks.

**Review by wborn — APPROVED** [review]
👍

**Pull request merged**